scripts/import-pr.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
```

Comment:
I think a nice feature will be to have the url of the original pr at the top of description so we can directly link to that 👍

Reply:
I can try that, should be easy enough to do in this PR.